fixtures/env.external

```diff
@@ -1,21 +1,17 @@
 # Matches fixtures/external_config.json
 
 AGENT_HOME=ethereum
+AGENT_REPLICA_0=polygon
```

Comment:
Can we have this var suffixed with the expected content. AGENT_REPLICA_0_NAME?

Reply:
If we do it should also be `AGENT_HOME_NAME` and `AGENT_REPLICA_{}_NAME`. Honestly think this is fine as is but up to you

Reply:
If it's easy to change my personal preference would be AGENT_HOME_NAME, if not I am ok to keep it that way.

Reply:
k changed